db.js

```diff
@@ -219,7 +219,7 @@ export class ActivityPubDB extends EventTarget {
     await tx.done()
   }
 
-  async searchNotes (criteria) {
+  async searchNotes (criteria, { skip = 0, limit = 10 } = {}) {
```

Comment:
Lets do a larger default. It's 32 elsewhere, maybe we can have it set as a const at the top like `DEFAULT_LIMIT`

Reply:
Yep.
I set the limit to 10 so that we can test the 'Load More' button.